var/spack/repos/builtin/packages/opendx/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class Opendx(AutotoolsPackage):
+    """Open Visualization Data Explorer."""
```

Comment:
:heart_eyes: That's vintage!

Reply:
Unfortunately it’s so vintage that it doesn’t compile on macOS or Linux